tests/unit/datarobot_drum/drum/test_args_parser.py

```diff
@@ -654,3 +659,16 @@ def test_read_triton_server_configs_success(
                 minimal_score_cmd_args,
                 is_env_variable,
             )
+
+
+class TestRuntimeParametersArgs:
+    @pytest.mark.parametrize("args", ["score_args", "server_args"])
+    def test_runtime_params_valid(self, request, args):
```

Comment:
nice!  please add for all of the positive cases

```
        CMRunnerArgsRegistry._reg_args_runtime_parameters_file(
            score_parser, perf_test_parser, server_parser, validation_parser
        )
```

Reply:
done